openlibrary/i18n/validators.py

```diff
@@ -1,15 +1,20 @@
-from babel.messages.catalog import TranslationError
+from itertools import groupby
+import re
+from typing import List
+
+from babel.messages.catalog import TranslationError, Message, Catalog
 from babel.messages.checkers import python_format
 
 
-def validate(message, catalog):
+def validate(message: Message, catalog: Catalog) -> List[str]:
     errors = _validate_fuzzy(message)
     errors.extend(_validate_format(message, catalog))
+    errors.extend(_validate_cfmt(message.id, message.string))
```

Comment:
Thanks to the type hints, I can now quickly/easily investigate what other options these have!

```suggestion
    if message.string and message.pluralizeable:
        errors.extend(_validate_cfmt(message.id, message.string))
```